src/Avalonia.Visuals/Visual.cs

```diff
@@ -239,6 +248,15 @@ protected IAvaloniaList<IVisual> VisualChildren
             private set;
         }
 
+        /// <summary>
+        /// Allows to explicitly force a rendering layer for a particular visual. Use with caution.
```

Comment:
A few things here:

- What does "Wants" mean here? Does it mean the request might not be respected? Because "wants" to me suggests a request that can be denied. If it will always be respected, maybe "ForceLayer" might be better?
- This is diverging from WPF/UWP's way of requesting a new layer. Are we OK with that?
- I think "Use with caution" should be in the `<Remarks>` section along with a description of _why_ it should be used with caution.

Reply:
> Does it mean the request might not be respected

It's not respected by ImmediateRenderer. The idea is that it's just a hint.

I think we could remove this change since it's not really needed for the purposes of this PR.